src/compiler/transformers/declarations.ts

```diff
@@ -233,7 +233,6 @@ function hasInternalAnnotation(range: CommentRange, currentSourceFile: SourceFil
     return stringContains(comment, "@internal");
 }
 
-/** @internal */
 export function isInternalDeclaration(node: Node, currentSourceFile: SourceFile) {

```

Comment:
One thing I'm wondering is if `sourceFile` should be optional here, sorta like other APIs in services. That and actually be named `sourceFile` for consistency.

Reply:
A consistent name would be nice.
Is sourceFile really optional in the body of the function? If so, consistency with similar functions would also be nice. But I'm not sure it's worth the effort to make the code resilient to its absence.

Reply:
It's not optional, I just know that many functions make it optional and then call `getSourceFileForNode` if not provided, such that if you have a `sourceFile` already, things are faster.

Reply:
Then both sound nice to have, but not requirements.

Reply:
I also now noticed that this is in `src/compiler/transforms/declaration` which probably is not the place to be exporting anything. Will have to move it.

But, I don't know if we even want to do this PR either. We didn't get to it in the design meeting.